_ml-commons-plugin/agents-tools/tools/connector-tool.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: default
+title: Connector tool
+has_children: false
+has_toc: false
+nav_order: 20
+parent: Tools
+grand_parent: Agents and tools
+---
+
+<!-- vale off -->
+# Connector tool
+**Introduced 2.15**
+{: .label .label-purple }
+<!-- vale on -->
+
+The `ConnectorTool` uses a [connector]({{site.url}}{{site.baseurl}}/ml-commons-plugin/remote-models/connectors/) to call any REST API function. For example, you can use a `ConnectorTool` to call a Lambda function through its REST API interface.
```

Comment:
If this refers to an AWS Lambda, function, we should use the full service name here and just "Lambda" thereafter.